src/frontend/screens/Quests/index.tsx

```diff
@@ -117,63 +127,100 @@ export function QuestsPage() {
       )
     }
   })
+  //Search Quest
+  const fuseOptions = {
+    keys: ['name', 'title'],
+    threshold: 0.2
+  }
+
+  const questsWithGameNames =
+    quests?.map((quest) => {
+      const gameName = listings?.[quest.project_id]?.project_meta?.name ?? ''
+      return {
+        ...quest,
+        title: gameName,
+        name: quest.name
+      }
+    }) ?? []
 
-  const gameTitleMatches = (quest: Quest) => {
-    const title = listings ? listings[quest.project_id]?.project_meta?.name : ''
-    return title?.toLowerCase().startsWith(searchText.toLowerCase())
+  const fuse = new Fuse(questsWithGameNames ?? [], fuseOptions)
+  const searchFilteredQuests = searchText
+    ? (fuse.search(searchText) as FuseResult<Quest>[])
+    : quests?.map((quest) => ({
+        id: quest.id,
+        project_id: quest.project_id,
+        name: quest.name,
+        title: listings?.[quest.project_id]?.project_meta?.name ?? '',
+        item: quest,
+        refIndex: 0
+      })) ?? []
```

Comment:
```suggestion
  let searchFilteredQuests = []
  if (searchText){
    searchFilteredQuests = fuse.search(searchText) as FuseResult<Quest>[]
  } else if (quests) {
    searchFilteredQuests = quests.map((quest) => ({
        id: quest.id,
        project_id: quest.project_id,
        name: quest.name,
        title: listings?.[quest.project_id]?.project_meta?.name ?? '',
        item: quest,
        refIndex: 0
      }))
  }
```

this is more readable